pages/authzed/concepts/restricted-api-access.mdx

```diff
@@ -237,14 +237,43 @@ policy:
 
 If you want to apply a static configuration to an existing SpiceDB cluster without downtime, you must conduct an upgrade process with the following steps:
 
-1. Create pre-shared keys that follow the token format for each client of your SpiceDB instance.
-   You should add those to your SpiceDB instance configuration.
-   You can do this by defining multiple PSKs via the ENV or flags as comma separated values.
-2. Update all your clients to use those new PSKs.
-3. Prepare the FGAM configuration YAML.
+1. Create pre-shared keys that follow the Restricted Access Token format for each client of your SpiceDB instance.
+   Using some Bash:
+
+   ```sh
+   # Generate your secret (substitute your preferred method for generating a cryptographically-secure random string here)
+   # This will be a part of the token
+   base64 < /dev/random | head -c64
+   # g2l2/YjC3jFg6FdV080qiqBPvCrlLuc9GcHutgHF4WhVjsg7+AvlqLmoCrJEC68t
+
+   # Hash that secret using sha256sum
+   # This will go in your FGAM configuration as the token hash
+   # NOTE: truncate the trailing spaces and "-". You just want the alphanum characters.
+   echo -n "g2l2/YjC3jFg6FdV080qiqBPvCrlLuc9GcHutgHF4WhVjsg7+AvlqLmoCrJEC68t" | sha256sum
+   # 1d619ac2f5013845c5f2df93add92fc87e88ca6c57d19a77d1b189663f1ff5b0  -
+   
+   # Add the prefix to create the token that you'll supply to your client
+   echo "sdbst_h256_g2l2/YjC3jFg6FdV080qiqBPvCrlLuc9GcHutgHF4WhVjsg7+AvlqLmoCrJEC68t"
+   # sdbst_h256_g2l2/YjC3jFg6FdV080qiqBPvCrlLuc9GcHutgHF4WhVjsg7+AvlqLmoCrJEC68t
+   ```
+
+2. Prepare the FGAM configuration YAML.
```

Comment:
should we put a sample yaml here?

Reply:
actually never mind, there is one above https://authzed.com/docs/authzed/concepts/restricted-api-access#static-configuration